### ISSUE TITLE
nvda input tables list doesn't contain an arabic tables

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3549]
**Reported by fatma.mehanna on 2013-09-25 11:53**
hi,
arabic braille sense users confirm that they can't read what is being typed on their braille display.
when i asked them to change the input table to arabic, they found out that the list of input tables doesn't include arabic.
note, the output list contains arabic tables.
so what should be done?
thanks

**Comment by feerrenrut** [comment]
We would happy to receive a pull request for this, for now I believe we are still blocked by the creation of Arabic translation tables (please correct me if I'm wrong).

**Comment by mohdshara** [comment]
Hi. An Arabic input table now exists, however it doesn't work very well, it can only type the Arabic characters that don't map to any English character. that's because the existing Arabic table defines English characters via an include statement. which I think is wrong and nonstandard. I tried to edit the table but it didn't work. It looks like simply editing the table  and restarting NVDA is not enough if you're not running from source. Can anyone help me with this quest?

**Comment by jcsteh** [comment]
Editing a table and restarting should be sufficient. If you're using an installed copy of NVDA, you'll have to edit it within the Program Files directory, which requires administrator privileges. It might be easier to use a portable copy for this reason.

**Comment by mohdshara** [comment]
thanks. I just can't find the table responsible for input C:\nvda\louis\tables\ar-ar-g1.utb seems to handle Arabic braille output. while ar.tbl doesn't seem to do anything. can you guide me in this regard?

**Comment by jcsteh** [comment]
The same tables are used for both input and output; in this case,
ar-ar-g1.utb. The .tbl files are just metadata and can be ignored; NVDA
does not use them.

**Comment by Hatoon-fel** [comment]
@jcsteh Hello!
I tried to edit the ar-ar-g1.utb to make it avoid confusion between Arabic and english characters, by trying to edit all include statements, But nothing change.
could you guide us to the right statements we have to edit, or if there is another file we need to edit please?
Thank you for all your efforts to make NVDA better!
@mohdshara

**Comment by jcsteh** [comment]
Please address questions/bugs regarding liblouis tables to the liblouis project. See https://github.com/liblouis/liblouis for details. There is a mailing list to which you can subscribe to ask questions. Thanks.

**Comment by jcsteh** [comment]
Closing because the table is now listed after #6449. It needs to be fixed to support back-translation properly, but that should be addressed in the liblouis project. Once the fixes make it into a liblouis release, this can be included in NVDA.
